puppet/zulip/manifests/profile/base.pp

```diff
@@ -45,6 +45,7 @@
         'curl',
         'wget',
         'jq',
+        'procps',
```

Comment:
This is arguably not needed, as systems that did not have `pgrep` could not have installed Zulip >= 1.4.0.  I added it for consistency.